packages/discord.js/src/util/Embeds.js

```diff
@@ -61,18 +61,22 @@ class Embeds extends null {
       url: data?.url,
       timestamp: data?.timestamp,
       color: data?.color,
-      footer: {
-        test: data?.footer?.text,
-        icon_url: data?.footer?.iconURL ?? data?.footer?.icon_url,
-      },
+      footer: data?.footer
+        ? {
+            test: data.footer.text,
```

Comment:
btw its text no test, thats a bug

Reply:
fixed (there was another issue in author)